src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -69,70 +69,71 @@ We recommend you start with the table at the beginning that shows high-level com
     </tr>
     <tr>
       <td>
-        Measure and track SLIs/SLOs
```

Comment:
Relocated this later in the list to keep APM things grouped together up top

Reply:
Sounds good!